gurobi_example.c

```diff
@@ -0,0 +1,144 @@
+/* Copyright 2019, Gurobi Optimization, LLC */
```

Comment:
Does any other file use this? This is here for just reference right?

Reply:
Yes, it's just for reference. Let's keep it for a bit until we get comfortable with our Rust API to Gurobi.